docs/en/sql-reference/aggregate-functions/reference/groupsortedarray.md

```diff
@@ -0,0 +1,48 @@
+---
+toc_priority: 108
+---
+
+# groupSortedArray {#groupSortedArray}
+
+Returns an array with the first N items in ascending order.
```

Comment:
Should the original column be sorted? Does it include duplicates in the final array?

Reply:
No, the original column can be sorted or not. 
Yes, it includes duplicates in the final array.